onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol

```diff
@@ -15,10 +15,10 @@ pragma solidity ^0.8.8;
 
 import {TestBase} from "../util/TestBase.sol";
 
-import {CartesiDApp} from "contracts/dapp/CartesiDApp.sol";
+import {CartesiDApp, VoucherReexecutionNotAllowed, EtherTransferFailed, OnlyDApp} from "contracts/dapp/CartesiDApp.sol";
 import {Proof} from "contracts/dapp/ICartesiDApp.sol";
 import {IConsensus} from "contracts/consensus/IConsensus.sol";
-import {OutputValidityProof, LibOutputValidation} from "contracts/library/LibOutputValidation.sol";
+import {OutputValidityProof, LibOutputValidation, IncorrectEpochHash, IncorrectOutputsEpochRootHash, IncorrectOutputHashesRootHash, InputIndexOutOfClaimBounds} from "contracts/library/LibOutputValidation.sol";
```

Comment:
This `import` statement is very lengthy. We should either split this into several lines or move these error definitions to inside the `LibOutputValidation` library, and access them from there.

Reply:
Custom errors **CAN** be inherited but **ARE NOT** accessible from the outside, second option does not seem viable. For the first one, it seems a very good idea but won't we have discrepancies each time we run prettier?

Reply:
> Custom errors CAN be inherited but ARE NOT accessible from the outside

I've made a small test locally, and it seems that you can indeed access errors defined in contracts. Like so:

```solidity
contract A {
  error FooError();
  
  function raiseFooError() external pure {
    revert FooError();
  }
}

contract ATest is Test {
  function testRaiseFoo() external {
    A a = new A();
    vm.expectEmit(A.FooError.selector); // <---
    a.raiseFooError();
  }
}
```

Reply:
We'll go trough changes in error definitions to allow this type of access in PR #54